### PR TITLE
MBL-11702 - show the eye if using new gradebook

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-11702
affects: Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: rn/Teacher/ios/Teacher/AppDelegate+Routes.swift [hunk 1]
```diff
@@ -71,6 +71,10 @@ extension AppDelegate {
             return WrongAppViewController.create(delegate: loginDelegate)
         })
 
+        HelmManager.shared.registerNativeViewController(for: "/courses/:courseID/assignments/:assignmentID/post_policy", factory: { _ in
+            return nil
```
Comment (bluwave) [inline rn/Teacher/ios/Teacher/AppDelegate+Routes.swift]: will add in PostPolicy native view controllers here in another pr

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1004242&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Teacher

[MBL-11702](https://instructure.atlassian.net/browse/MBL-11702)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 95.05% | 95.01% | 0.04%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request closed without merging**